conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -12,88 +12,61 @@
 import com.bitmovin.player.api.media.video.quality.VideoQuality;
 import com.bitmovin.player.api.source.Source;
 import com.bitmovin.player.api.source.SourceConfig;
-import com.conviva.api.AndroidSystemInterfaceFactory;
-import com.conviva.api.Client;
-import com.conviva.api.ClientSettings;
-import com.conviva.api.ContentMetadata;
-import com.conviva.api.ConvivaException;
-import com.conviva.api.SystemFactory;
-import com.conviva.api.SystemSettings;
-import com.conviva.api.player.PlayerStateManager;
-import com.conviva.api.system.SystemInterface;
+import com.conviva.sdk.ConvivaAdAnalytics;
+import com.conviva.sdk.ConvivaAnalytics;
+import com.conviva.sdk.ConvivaSdkConstants;
+import com.conviva.sdk.ConvivaVideoAnalytics;
 
 import java.util.HashMap;
 import java.util.Map;
 
-public class ConvivaAnalytics {
+public class ConvivaAnalyticsIntegration {
 
-    private static final String TAG = "ConvivaAnalytics";
+    private static final String TAG = "ConvivaAnalyticsInt";
 
-    private Client client = null;
     private Player bitmovinPlayer;
     private ContentMetadataBuilder contentMetadataBuilder = new ContentMetadataBuilder();
     private ConvivaConfig config;
-    private int sessionId = Client.NO_SESSION_KEY;
-    private PlayerStateManager playerStateManager;
-    private ClientMeasureInterface clientMeasureInterface;
+    private ConvivaVideoAnalytics convivaVideoAnalytics;
+    private ConvivaAdAnalytics convivaAdAnalytics;
```

Comment:
`ConvivaAdAnalytics` object is only created but is not used anywhere in the code? Should this be used for providing Ad related metadata?

Reply:
ok, for my clarity, do you mean that Bitmovin team did not request for Ad reporting to be supported in this upgrade  and was supposed to be taken up as a follow-up change? Sorry for asking but I may be missing some information about the scope:)

Reply:
Nothing specific was discussed here, but the focus was of course on maintaining the same level of reporting as in the previous version of the SDK. The changes in the latest version handle some of the ad related stuff automatically, but I will check again that we are not missing anything that was available previously.

Reply:
We are not missing anything, and I suggest that we implement Ad Analytics at a later date.

Reply:
ok, thanks. To be sure I understood correctly, the updated implementation is at par with existing implementation for Ad reporting, correct?

Reply:
Yes, that's correct. Tried and tested in touchstone too.